### ISSUE TITLE
Cannot

### ISSUE THREAD

**Issue by NinaMercer** [TES5Edit/TES5Edit#1073]
**What is the version of xEdit you are using?**
4.0.4

**Describe the bug**
Simply doesn't work.
"Using Skyrim Special Edition Data Path: E:\program files (x86)\steam\steamapps\common\Skyrim Special Edition\Data\
Using Backup Path: E:\program files (x86)\steam\steamapps\common\Skyrim Special Edition\Data\SSEEdit Backups\
Using Scripts Path: C:\Users\spook\Documents\SSEEdit 4.0.4\Edit Scripts\
Using Cache Path: E:\program files (x86)\steam\steamapps\common\Skyrim Special Edition\Data\SSEEdit Cache\
Using ini: C:\Users\spook\AppData\Local\Temp\My Games\Skyrim Special Edition\Skyrim.ini
Fatal: Could not find ini"

**To reproduce**
1. Install XEdit
2. Add to MO2
3. Click run

**Expected behavior**
It to show me my plugins and whatnot.

**Screenshots**
![image](https://user-images.githubusercontent.com/61397682/216297864-b6005657-5cdc-4845-9817-b24ef9c07276.png)

**Additional context**
Looks like a dev had a brainfart and set the wrong path.

**Comment by ElminsterAU** [comment]
F*** you very much.

Have you ever considered that if a program works successfully for literally millions of other people, the problem might be related to you and your setup instead of the program?

**Comment by NinaMercer** [comment]
Perhaps, but i've googled the issue, and a few other people have encountered it. You could(not sure if easily), let the user set the ini path themselves, rather than set it to what you expect.

**Comment by NinaMercer** [comment]
Though, it appears this was actually just Windows Defender not liking XEdit anyways.

**Comment by robertgk2017** [comment]
How about you actually set up the program correctly according to the instructions.

**Comment by FelesNoctis** [comment]
> How about you actually set up the program correctly according to the instructions.

My first thought was piracy since xEdit checks all typical installation paths for the various distribution platforms. Of course that's assuming they're actually trying to run xEdit for SSE, and haven't just used the wrong naming convention (say for FO4 or SLE). They'd also need to run SSE through the launcher once if it's a fresh install. So I mean, a lot could be going on there.

> Perhaps, but i've googled the issue, and a few other people have encountered it.

That would be the exception, not the norm. There are many reasons why xEdit searches for the known valid paths itself, and it very rarely if ever causes a problem.

> Though, it appears this was actually just Windows Defender not liking XEdit anyways.

I don't think I've ever had Windows Defender interfere with xEdit, but that's my experience. If that's the case and you figured it out, great! What was the solution? That would help other people in the future. @NinaMercer

**Comment by NinaMercer** [comment]
I tried running XEdit over and over again, repeatedly. Until finally, i got a notification from Windows Defender. I clicked it, and apparently it thought XEdit was ransomware for some reason. I can't remember what the button said exactly, but there was a button that said either "unblock" or "allow". I simply clicked that, and tried running XEdit again, and it worked fine. It found the ini, and i was able to select what plugins i wanted to edit. I guess Windows 10 just doesn't have WD's ransomware protection on by default or something? I can't remember.

But yeah, you just have to allow XEdit through the built-in ransomware protection, if you encounter this issue.
